modules/update-preparing-evaluate-apirequestcount.adoc

```diff
@@ -24,15 +24,12 @@ $ oc get apirequestcounts
 .Example output
 [source,terminal]
 ----
-NAME                                                                      REMOVEDINRELEASE   REQUESTSINCURRENTHOUR   REQUESTSINLAST24H
+NAME                                                                 REMOVEDINRELEASE   REQUESTSINCURRENTHOUR   REQUESTSINLAST24H
 ...
-flowschemas.v1beta1.flowcontrol.apiserver.k8s.io                          1.26               0                       16
-flowschemas.v1beta2.flowcontrol.apiserver.k8s.io                                             101                     857
-groups.v1.user.openshift.io                                                                  22                      201
-hardwaredata.v1alpha1.metal3.io                                                              3                       33
-helmchartrepositories.v1beta1.helm.openshift.io                                              142                     628
-horizontalpodautoscalers.v2.autoscaling                                                      11                      103
-horizontalpodautoscalers.v2beta2.autoscaling                              1.26               0                       15
+csistoragecapacities.v1.storage.k8s.io                                                  14                      380
+csistoragecapacities.v1beta1.storage.k8s.io                          1.27               0                       16
```

Comment:
@sanchezl I didn't have this in my test cluster, so I put in this line to what I think it would look like, can you confirm whether this is right or not?